content/applications/hr/payroll/payslips.rst

```diff
@@ -0,0 +1,402 @@
+detail page, modifications can be made. To make any modifications to a batch, click the
+:guilabel:`Edit` button. Then, proceed to make any necessary changes.
+
+.. image:: payslips/edit-batch.png
+   :align: center
+   :alt: Edit a batch.
+
+When all desired changes have been made, click either :guilabel:`Save` to save the changes, or
+:guilabel:`Discard` to revert back to the original data.
+
+After modifications have been saved, click the :guilabel:`Generate Payslips` button to reveal a
+:guilabel:`Generate Payslips` pop-up form, in which payslips affected by the changes can be created
+(or modified).
+
+All payslips associated with the batch are displayed in the employees section of the
+:guilabel:`Generate Payslips` pop-up. To filter the results by either the
+:guilabel:`Salary Structure` and/or the :guilabel:`Department`, select a salary structure and/or
+department from the respective drop-down menus.
+
+Only employees that match the selected :guilabel:`Salary Structure` and/or :guilabel:`Department`
+appear in the employees list. Click the :guilabel:`Generate` button at the bottom of the
+:guilabel:`Generate Payslips` pop-up window to generate the modified payslips, and close the pop-up
+window.
+
+.. image:: payslips/generate-payslips-batch.png
+   :align: center
+   :alt: Generate payslips from the edited batch.
+
+Back on the batch details page, click the :guilabel:`Create Draft Entry` smart button to create a
+draft of the payslips.
+
+.. image:: payslips/draft-from-batch.png
+   :align: center
+   :alt: Generate payslips from the edited batch.
+
+Once the draft payslips are created, the button changes to say :guilabel:`Make Payment`. Click the
+:guilabel:`Make Payment` button. A pop-up window appears, in which the bank journal information must
+be entered. Select the :guilabel:`Bank Journal` from the drop-down list, and click
+:guilabel:`Confirm` to process the payslips, and pay the employees.
+
+On the batch detail page, the number of payslips in the batch is accessible via the
+:guilabel:`Payslips` smart button, located in the top-right corner. Individual payslips for the
+batch can be viewed by clicking the :guilabel:`Payslips` smart button in the top-right corner.
+
+Use the breadcrumb menu to navigate back to the individual batch detail page, or back to the list of
+all batches.
+
+.. image:: payslips/payslip-batches.png
+   :align: center
+   :alt: Click the Payslips smart button to view the ndividual payslips in the batch.
+
+Create a new batch
+------------------
+
+To create a new batch of payslips from the :guilabel:`Payslips Batches` page
+(:menuselection:`Payroll app --> Payslips --> Batches`), click the :guilabel:`Create` button in the
+upper-left corner. Doing so reveals a blank batch payslip detail form on a separate page.
+
+.. image:: payslips/create-batch.png
+   :align: center
+   :alt: Click the Create smart button to create a new batch.
```

Comment:
I think your instructions are clear enough that this image is not necessary, can remove :)
```suggestion
```